clouds/tests/aws/test_aws_utils.py

```diff
@@ -0,0 +1,148 @@
+import pytest
+from typing import Dict, List
+
+from clouds.aws.aws_utils import (
+    set_and_verify_aws_credentials,
+    set_and_verify_aws_config,
+    delete_all_objects_from_s3_folder,
+    delete_bucket,
+    aws_region_names,
+    get_least_crowded_aws_vpc_region,
+)
+
+OS_ENVIRON_PATCH_STR: str = "clouds.aws.aws_utils.os.environ"
+CONFIG_PARSER_PATCH_STR: str = "clouds.aws.aws_utils.ConfigParser"
+EC2_CLIENT_PATCH_STR: str = "clouds.aws.aws_utils.ec2_client"
+BOTOCORE_CLIENT_PATCH_STR: str = "clouds.aws.aws_utils.botocore.client"
+
+DUMMY_BUCKET_NAME: str = "test_bucket"
+US_EAST_REGION_STR: str = "us-east-1"
+US_WEST_REGION_STR: str = "us-west-1"
+DEFAULT_AWS_CONFIG_SECTION: str = "default"
+
+
+@pytest.fixture
+def mock_os_environ(mocker):
+    os_environ_mock = mocker.patch(OS_ENVIRON_PATCH_STR)
+    return os_environ_mock
+
+
+@pytest.fixture
+def mock_config_parser(mocker):
+    return mocker.patch(CONFIG_PARSER_PATCH_STR)
+
+
+@pytest.fixture
+def mock_config_parser_instance(mock_config_parser, mocker):
+    mock_config_parser_instance = mocker.MagicMock()
```

Comment:
please rename mock_config_parser_instance so the var will not have the same name as the func

Reply:
done